bin/git-delete-squashed.js

```diff
@@ -25,23 +26,27 @@ function git (args) {
   }).then(stdout => stdout.replace(/\n$/, ''));
 }
 
+if (process.argv.length > 0) {
+  selectedBranchName = process.argv[1];
+}
```

Comment:
Instead of `selectedBranchName || DEFAULT_BRANCH_NAME` everywhere, you could've just

```js
const selectedBranchName = process.argv[0] || DEFAULT_BRANCH_NAME;
```

or

```js
} else {
  selectedBranchName = DEFAULT_BRANCH_NAME;
}
```

Reply:
I'm on Windows so that changes the order of arguments because I have to run the script with node.exe. Let me test on Mac and Linux to see what my updates will do.